crates/task-impls/src/network.rs

```diff
@@ -522,6 +531,20 @@ impl<
                     .await;
                 None
             }
+            HotShotEvent::VidRequestSend(req, signature) => Some((
+                req.key.clone(),
+                MessageKind::<TYPES>::from_consensus_message(SequencingMessage::General(
+                    GeneralConsensusMessage::VidRequested(req.clone(), signature),
+                )),
+                TransmitType::DaCommitteeBroadcast,
```

Comment:
i dont know if we want to broadcast to whole committee or do it like we did before, choose a member of the committee for a view, if no response and it times out try another member

Reply:
For now, I don't necessarily see an issue with sending to the entire committee. I think doing a round-robin request-response the right way will require more work than is probably needed for this PR.